packages/mco/components/mco-dashboard/data-policy/dr-dashboard.tsx

```diff
@@ -35,8 +40,19 @@ export const DRDashboard: React.FC = () => {
     basePath: ACM_ENDPOINT,
     cluster: HUB_CLUSTER_NAME,
   });
-  const [argoApplicationSetResources, loaded, loadError] =
-    useArgoApplicationSetResourceWatch();
+  const [managedClusters, clustersLoaded, clusterLoadError] =
+    useACMManagedClusterWatch();
+  const drResources = useDisasterRecoveryResourceWatch();
+  const [formattedDRResource, drLoaded, drLoadError] =
+    useDisasterRecoveryFormatter({ ...drResources });
```

Comment:
do not destructure...
```suggestion
    useDisasterRecoveryFormatter(drResources);
```

Reply:
ack